src/main/java/lotto/domain/price/PurchaseInfo.java

```diff
@@ -0,0 +1,47 @@
+package lotto.domain.price;
+
+import lotto.domain.ticket.LottoTickets;
+import lotto.exception.AvailableCountExceedException;
+import lotto.exception.ErrorMessage;
+
+public class PurchaseInfo {
+
+    private Price price;
+    private LottoTickets manualTickets;
+
+    private PurchaseInfo(final Price price, final LottoTickets manualTickets) {
+        validatePrice(price);
+        verifyAvailableCount(price, manualTickets.count());
+
+        this.price = price;
+        this.manualTickets = manualTickets;
+    }
+
+    public static PurchaseInfo valueOf(final Price price, final LottoTickets manualTickets) {
+        return new PurchaseInfo(price, manualTickets);
+    }
+
+    private static void validatePrice(final Price price) {
+        if (price == null) {
+            throw new IllegalArgumentException(ErrorMessage.NULL_VALUE);
+        }
+    }
+
+    private static void verifyAvailableCount(final Price price, final int manualTicketCount) {
+        if (price.isExceedCount(manualTicketCount)) {
+            throw new AvailableCountExceedException();
+        }
+    }
+
+    public int getManualTicketsCount() {
+        return manualTickets.count();
+    }
+
+    public int getAutoTicketsCount() {
+        return price.ticketCount() - manualTickets.count();
```

Comment:
```suggestion
        return price.ticketCount() - getManualTicketsCount();
```
getManualTicketsCount() 를 이용하면 추후에 변경에 좀 더 영향을 덜 받을 수 있지 않을까요?